packages/std/src/imports.rs

```diff
@@ -13,7 +16,10 @@ use crate::types::{CanonicalAddr, HumanAddr};
 static MAX_READ: usize = 2000;
 
 // this should be plenty for any address representation
-static ADDR_BUFFER: usize = 72;
+static ADDR_BUFFER: usize = 100;
```

Comment:
There is a global max length of `90` specified for bech32 strings: https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#bech32 So HRP+separator+data combined must ensure to not exceed this. This would be a good limit and source the decision here.